docs/docs/integrations/analytics/adobe.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Adobe Analytics Integration
+sidebar_label: Adobe Analytics
+hide_title: true
+---
+
+![Adobe](/img/integrations/adobe/adobe-logo.svg)
+
+You can integrate Flagsmith with Adobe Analytics. You can automate AB tests by connecting the Flagsmith platform with
+Adobe Analytics.
+
+:::tip
+
+The Adobe Analytics integration is in Beta - please contact us to join the Beta!
```

Comment:
I wonder if we should be less aggressive here, could we say `alpha` ?

Reply:
OK Yep updated.